worklist/views_helper_functions/fetch_articles_from_petscan.py

```diff
@@ -12,7 +12,7 @@ def fetch_articles_from_petscan(psid):
               'using ID: {0}'.format(psid)
     articles = []
     parameters = {'format': 'json', 'psid': psid}
-    url = "https://petscan.wmflabs.org"
+    url = PETSCAN_URL
```

Comment:
Since you've made it as a constant directly use it in the function. No need of making an extra variable.

Reply:
Thanks, I have made the change.